x/attribute/client/cli/tx.go

```diff
@@ -222,3 +229,14 @@ func NewDeleteAccountAttributeCmd() *cobra.Command {
 
 	return cmd
 }
+
+func addExpiresFlagCmd(cmd *cobra.Command) {
+	cmd.Flags().String(FlagExpires, "", "expiration period (e.g. '1y')")
+}
+
+// parseExpires checks expires flag, else uses the default expiration period
+func parseExpires(cmd *cobra.Command) string {
+	flagSet := cmd.Flags()
+	expires, _ := flagSet.GetString(FlagExpires)
+	return expires
+}
```

Comment:
These should be public in the expiration module. Also, the description should have more details. Specifically, I'd like to see all the options present.